zerver/lib/actions.py

```diff
@@ -2721,9 +2721,6 @@ def check_update_message(
     if not user_profile.realm.allow_message_editing:
```

Comment:
Can you extend the docstring to say that it expects validate_message_edit_payload to have been called to validate that the payload is not malformed?